src/hooks/hotelsLoader.js

```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line no-unused-vars
+import React, { useEffect, useState } from "react";
+
+export const hotelsLoader = () => {
```

Comment:
Small thing, all hooks should start from `use` prefix
```suggestion
export const useHotelsLoader = () => {
```